src/common/_modules/global/modules/dns/dns_io_pagopa_it.tf

```diff
@@ -146,3 +146,13 @@ resource "azurerm_dns_a_record" "ipatente_io_pagopa_it" {
 
   tags = var.tags
 }
+
+# ipatente.io.pagopa.it
+resource "azurerm_dns_ns_record" "ipatente_io_pagopa_it_ns" {
```

Comment:
Why do you need `ns` record?

Reply:
It looks like this is not supported via terraform neither via api. Only via ARM template or portal
https://github.com/hashicorp/terraform-provider-azurerm/issues/14733#issuecomment-1002214173